### PR TITLE
Makes roboticists not get engineering tape in their belts

### PR DESCRIPTION
**Pull request by bruhlookatthisdood**
# Document the changes in your pull request

Gives the roboticists their own toolbelt type, normal utility belt but without the engineering tape, because why would they need it

# Wiki Documentation
no
# Changelog

Edit the changelog below to reflect the changes made by this PR, even if the changes are minor - required for every PR.
If you add a name after the ':cl', that name will be used in the changelog. Leave it empty to use your GitHub name.

:cl:  
tweak: roboticists no longer get tape in their toolbelts.
/:cl:

### CODE REVIEW

**Comment by Spacemanspark** [comment]
Unneeded nerf, now I can't grief Malice by surrounding them with tape >:(

**Comment by bruhlookatthisdood** [comment]
> Unneeded nerf, now I can't grief Malice by surrounding them with tape >:(

Science Tape?

File: code/game/objects/items/storage/belt.dm [hunk 1]
```diff
@@ -108,6 +108,7 @@
 	new /obj/item/extinguisher/mini(src)
 	new /obj/item/holosign_creator/multi/CE(src)
 
+
```
Comment (JamieD1) [inline code/game/objects/items/storage/belt.dm]:
```suggestion
```

**Review by JamieD1 — CHANGES_REQUESTED** [review]
I would prefer you made an engineering subtype instead and moved the tape to that.

**Review by JamieD1 — APPROVED** [review]

**Pull request merged**